build.gradle

```diff
@@ -21,11 +21,12 @@ buildscript {
     ext.kotlin_version = '1.4.0'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```

Comment:
Have you checked that all of the dependencies previously available on `jcenter` are now on `mavenCentral`? I know of a few deps that haven't been migrated yet, so I want to ensure they are all available before making the switch here.

Reply:
See the other comment